test/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+# Each source test file is instantiated in single and double precision
+set(TESTS basicpassfail dumbinputs finufft1d_test finufft1dmany_test finufft2d_test finufft2dmany_test finufft3d_test finufft3dmany_test)
```

Comment:
weirdly when I run `ctest` after building tests, the basicpassfail{f} ones built but not tested. There are 15 tests. Any ideas why?

Reply:
Thanks, I didn't see because I only looked into `checkfinufft.sh`. Added to the list of tests.